R/get_gsw.R

```diff
@@ -1,3 +1,90 @@
+#' Global Surface Water Change
+#'
+#' The change in water occurrence intensity between the two periods is derived
+#' from homologous pairs of months (i.e. same months containing valid
+#' observations in both periods). The difference in the occurrence of surface
+#' water was calculated for each homologous pair of months. The average of all
+#' of these differences constitutes the Surface Water Occurrence change
+#' intensity. The raster files have integer cell values between \code{[0, 200]}
+#' where 0 represents surface water loss and 200 represents surface water gain.
+#'
+#' @name global_surface_water_change
+#' @docType data
+#' @keywords resource
+#' @format Global raster layers
```

Comment:
Is this one global raster layer? For esalandcover we write: "A global tiled raster resource available for years 2015 to 2019." Could you please adjust?